sample/src/main/java/kin/core/sample/OnBoarding.java

```diff
@@ -19,10 +19,10 @@
 
 class OnBoarding {
 
-    private static final String URL_CREATE_ACCOUNT = "http://friendbot-kik.kininfrastructure.com/?addr=";
+    private static final String URL_CREATE_ACCOUNT = "http://friendbot-playground.kininfrastructure.com/?addr=";
     private static final int FUND_KIN_AMOUNT = 6000;
     private static final String URL_FUND =
-        "http://159.65.84.173:5000/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
+        "http://faucet-playground.kininfrastructure.com/fund?account=%s&amount=" + String.valueOf(FUND_KIN_AMOUNT);
```

Comment:
@yosriz why do the client needs it?

Reply:
for sample app, sample app has an onboarding example that uses friendbot/facuet to initiate account

Reply:
if this is only the for the sample app and this should not be exposed as part of the sdk maybe better write it hardcoded in the sample app?

Reply:
it is hardcoded in the sample app, that's part of the sample app